creator-onboarding/artist-onboarding-steps.md

```diff
@@ -2,64 +2,68 @@
 description: >-
   Below you'll find an overview of each step of our onboarding process as well
   as an **estimated** time required for each step.
-  
+order: 1000
 ---
 
 # Artist Onboarding Steps
 
+!!!info Timeline
+Due to the surge in Art Blocks popularity, applications may take up to several months to process. Applications will be reviewed in the order they were submitted.
+!!!
+
 ## 1. Application
 
-During one of our open application periods, submit a prototype of a proposed project along with your contact information and creative history. Our team will review each submission and reach out to you to get an updated version of your working prototype when it's your turn in the queue.&#x20;
+During one of our open application periods, submit a prototype of a proposed project along with your contact information and creative history. Our team will review each submission and reach out to you to get an updated version of your working prototype when it's your turn in the queue.
 
-_Timeline: Due to the surge in Art Blocks popularity, applications may take up to several months to process. Applications will be reviewed in the order they were submitted._
+[!badge Timeline: Up to several months]
```

Comment:
a bit confused. what's the recommendation for trimming?

Reply:
@geoffreymcgill could you elaborate on why you wanted to trim here?

Reply:
I was trying to use a `[!badge]` component for each of those `Timeline:` notes, but the first one was extra long and didn't really work as a badge. So, I moved the full msg above the `1. Application` section and recreated as an [alert](https://retype.com/components/alert/) component, then tightened up the badge to only `Up to several months`.

I just pushed an update to the PR reversing the change and using an alert component instead of badge. You can see the new page at:

https://geoffreymcgill.github.io/artblocks-docs/creator-onboarding/readme/artist-onboarding-steps/

The `Timeline:` is not _exactly_ consistent amongst all the step, but still looks good and highlights those timeline messages nicely. 

Hope this helps.